.github/workflows/test.yml

```diff
@@ -239,8 +239,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macOS-latest, windows-2019, ubuntu-22.04]
-        ros_distribution: [humble, iron, rolling]
+        include:
+          - os: ubuntu-22.04
+            ros_distribution: humble
+          - os: ubuntu-22.04
+            ros_distribution: iron
+          - os: ubuntu-24.04
+            ros_distribution: rolling
+          - os: windows-2019
+            ros_distribution: humble
+          - os: windows-2019
+            ros_distribution: iron
+          - os: windows-2019
+            ros_distribution: rolling
+          - os: macOS-latest
+            ros_distribution: humble
+          - os: macOS-latest
+            ros_distribution: iron
+          - os: macOS-latest
+            ros_distribution: rolling
```

Comment:
@christophebedard AFAIK we don't support macOS officially anymore.  
Can we remove these macOS related items?

Reply:
See https://github.com/ros-tooling/action-ros-ci/issues/845